anvil/free_scalar.py

```diff
@@ -140,7 +149,7 @@ def gen_complex_normal(n_sample, sigma, real=False):
         Returns
         -------
         out: numpy.ndarray
-            complex array of shape (n_sample, *sigma.shape)
+            complex array of shape (n_sample, ``*`` sigma.shape)
```

Comment:
This doesn't look good in docs

```suggestion
            complex array of shape: ``(n_sample, *sigma.shape)``
```

Reply:
> I think in general, when making a change like this which refactors a fair amount of code, it might be good to get a consensus before making the changes rather than opening a PR over the weekend with a fair amount of executive decisions. Because now I'm faced with rejecting something that was clearly a non-negligible amount of work or accepting something which I'm not a huge fan of. Anyway, my opinion isn't that strong and I don't think we want to waste time discussing this, so I'm happy to merge this once you've addressed the comments.

You're right, I apologise for that. The rationale was that I was sticking to a PEP standard and that it wouldn't matter if type hints were in the signature or the docstring since Napolean could be configured to display them either way, but after realising that this would cause issues with reportengine I just decided to continue writing docstrings rather than go back and 'fix' all of the typehints. I'm more than happy to do that and it's really not too much work.

Happy to make these changes and will do so now.

Reply:
> I'm more than happy to do that and it's really not too much work.

nah, I think there are better things for you to do, and this is really just a personal flavour. If nothing is broken then this is fine.

At some point it might be worth adding a few regression style tests on the observables - just to make sure they run as expected and produce similar results given the same model or something but again we can do this after submission (and perhaps I should do it to better understand some observables...)